### PR TITLE
[TorchAcc] Update padding strategy when using persistent cache

### PR DESCRIPTION
**Pull request by eedalong**
# PR type
- [ ] Bug Fix
- [x] New Feature
- [ ] Document Updates
- [ ] More Models or Datasets Support

# PR information

Optimize padding strategy when persistent cache is enabled, so we can enjoy performance boost with little extra compilaiton.

## Experiment results

Around 10% e2e performance improvement for TorchAcc backend.

### CODE REVIEW

File: swift/torchacc_utils.py [hunk 1]
```diff
@@ -30,6 +30,15 @@ def get_bucket_sizes(max_length: int) -> List[int]:
     if os.getenv('TORCHACC_DATA_BUCKETS') is not None:
         bucket_sizes = [int(x) for x in os.getenv('TORCHACC_DATA_BUCKETS').split(',')]
         bucket_sizes.append(max_length)
+    elif os.getenv('TORCHACC_CACHE_PATH') is not None: # padding strategy when persistent cache is enabled
+        p = 1.4
```
Comment (anw90) [inline swift/torchacc_utils.py]: Maybe we could add an environment variable for the p.

File: swift/torchacc_utils.py [hunk 2]
```diff
@@ -30,6 +30,15 @@ def get_bucket_sizes(max_length: int) -> List[int]:
     if os.getenv('TORCHACC_DATA_BUCKETS') is not None:
         bucket_sizes = [int(x) for x in os.getenv('TORCHACC_DATA_BUCKETS').split(',')]
         bucket_sizes.append(max_length)
+    elif os.getenv('TORCHACC_CACHE_PATH') is not None: # padding strategy when persistent cache is enabled
```
Comment (anw90) [inline swift/torchacc_utils.py]: Could we replace the `else` block with this logic as defualt bucketing strategy?

File: swift/torchacc_utils.py (hunk 1 above)
Comment (eedalong) [reply swift/torchacc_utils.py]: Fixed.

File: swift/torchacc_utils.py (hunk 2 above)
Comment (eedalong) [reply swift/torchacc_utils.py]: Fixed.

**Review by anw90 — COMMENTED** [review]
LGTM

**Review by anw90 — APPROVED** [review]

**Pull request merged**